content/community/contributing/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "How to contribute"
+autotoc: false
+---
+<div class='right'><img src="/images/undraw-illustrations/contribute.svg" alt="" width="240" /></div>
+
+If you want to contribute to the Galaxy Project, there are plenty of ways to do it. And here is the good news: __you can contribute with code, but not only with code__!
+
+## Contribute with __code__
+- [__Galaxy Project GitHub Organization__](https://github.com/galaxyproject): repositories of the Galaxy ecosystem, including the [main code of Galaxy](https://github.com/galaxyproject/galaxy).
+
+- __How?__ 
+  - Start by looking at the _CONTRIBUTING.md_ (or _CONTRIBUTING.rst_) file in the repository of your interest.
+  - There are [__Working Groups__](https://galaxyproject.org/community/wg/) focused on particular parts of the ecosystem, you can reach out to them if you want to help.
```

Comment:
_**if you want to know more about how to help**_ instead of  _**if you want to help**_